CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+# Contributing to farmOS
+
+There are many ways to get involved in the farmOS community. You don't need to
+be a programmer. All are welcome to take part in the process.
+
+## Use farmOS
+
+First and foremost: use farmOS.
+
+Building a critical mass of farmOS users is the best way for the project to
+grow and thrive. If you would like to improve your record keeping habits, or
+just want a place to put everything so you can access it easily, farmOS can
+help. If you have a social media presence, or if you know farmers - help tell
+the world about farmOS!
+
+## Questions, Feature requests, Bug reports, etc
+
+If you have questions, ideas for new features, or if you find bugs, you can
+can start a public thread to discuss/develop/debug them.
+
+*farmOS comprises multiple projects and modules, across different codebases,
+each with their own issue queues and roadmaps, so if you are unsure where to
+put something, follow these guidelines.*
+
+**Questions**, **support requests**, and **general discussions** should be in
+the [farmOS Forum](https://farmOS.discourse.group).
+
+**Bug reports** can be posted to the
+[farmOS issue queue on GitHub](https://github.com/farmOS/farmOS/issues).
+Issues in this queue will be triaged and moved or copied into the appropriate
+issue queues for the code responsible for them.
+
+**Feature requests** should start in the forum, most of the time. Developers
+and other community members can discuss the idea and decide where it makes
+sense to build it. In many cases, new features can be built as standalone
+modules or apps, and don't need to be part of the farmOS code itself. This
+modular and API-driven approach to development allows for experimentation and
+innovation in the "contrib" space. If core farmOS features are built, issues
+for each specific piece can be created in the appropriate project issue queues
+and added to their respective roadmaps.
```

Comment:
> If core farmOS features are built, issues for each specific piece can be created in the appropriate project issue queues
and added to their respective roadmaps.

Seems a little unclear why the assignment of issues is predicated on "core farmOS features [being] built". Shouldn't this say something more like; "**If the feature request requires core farmOS feature(s) to be built**, issues for for each specific piece can be created..."?